### DISCUSSION TITLE
Tabs in a form [Oct 06]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4067] [Discord Threads]
> 🙋‍♂️ **HugoBarjot**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@256738446071169026> Tabs in form is not supported. New feature available soon will be `Survey form`  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1024925747216928778)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
